facebook-commerce.php

```diff
@@ -368,6 +368,14 @@ public function __construct( WC_Facebookcommerce $facebook_for_woocommerce ) {
 
 		add_action( 'untrashed_post', [ $this, 'fb_restore_untrashed_variable_product' ] );
 
+		// Ensure product is deleted from FB when moved to draft.
+		add_action(
+			'transition_post_status',
+			array( $this, 'delete_draft_product' ),
+			10,
+			3
+		);
```

Comment:
I noticed that a few lines above, we have this action hook: https://github.com/woocommerce/facebook-for-woocommerce/blob/cccb9f88682c57166fa1b6709f19cc987fea7946/facebook-commerce.php#L367

They're doing the same thing. I'd remove that if it's not working, and I am not sure there's value in keeping both codes.

Reply:
I don't understand this comment 😓 

One hook is for deleting, one if for trashing and one is for changing to draft :) Which one should we delete? Do I understand this correctly?

Reply:
We need to keep both. The new action hook only deletes the product if it is changed to draft. The existing one `add_action( 'wp_trash_post', [ $this, 'on_product_delete' ] ); ` deletes the product if it is moved to trash.

Reply:
Had a bit of a brain fart here. Draft and trash aren't the same. You're both correct. I will resolve the threat.